jsons/TileResources.json

```
@@ -214,7 +214,102 @@
 		"resourceType": "Strategic",
 		"uniques": ["Cannot be traded","Not shown on world screen"],
 	},
-	
+	// Ability Cards
+	{
+		"name": "Blank Card",
+		"resourceType": "Luxury",
+		"uniques": ["Not shown on world screen"]
+	},
+	{
+		"name": "Yin-Yang Orb Card",
+		"resourceType": "Luxury",
+		"uniques": ["Not shown on world screen","[+8]% Strength <for [All] units> <with [Yin-Yang Orb Card]>"]
+	},
+	{
+		"name": "Mini-Hakkero Card",
+		"resourceType": "Luxury",
+		"uniques": ["Not shown on world screen","[+12]% Strength <for [Siege] units> <with [Mini-Hakkero Card]>"]
+	},
+	{
+		"name": "Shanghai Doll Card",
+		"resourceType": "Luxury",
+		"uniques": ["Not shown on world screen","[1] free [Worker] units appear <with [Shanghai Doll Card]>"]
+	},
+	{
+		"name": "Back Door Card",
+		"resourceType": "Luxury",
+		"uniques": ["Not shown on world screen","[+15]% Strength <when defending> <when adjacent to a [{Friendly} {Military}] unit> <for [Military] units> <with [Back Door Card]>"]
+	},
+	{
+		"name": "Offerings to a Sacred Mountain Card",
+		"resourceType": "Luxury",
+		"uniques": ["Not shown on world screen","[+1 Gold] from [Mountain] tiles [in all cities]","[+1] Sight <for [Scout] units> <with [Offerings to a Sacred Mountain Card]>"]
+	},
+	{
+		"name": "Sutra of Dharmatic Power Card",
+		"resourceType": "Luxury",
+		"uniques": ["Not shown on world screen","[+33]% Strength <when attacking> <when below [10] HP> <with [Sutra of Dharmatic Power Card]>"]
```

This will rarely trigger, it would be best if it's `<when below [30] HP>` instead
```suggestion
		"uniques": ["Not shown on world screen","[+33]% Strength <when attacking> <when below [30] HP> <with [Sutra of Dharmatic Power Card]>"]
```

---

ok